src/formio/components/Children/AddChildModal.stories.ts

```diff
@@ -3,7 +3,7 @@ import {fn} from '@storybook/test';
 
 import {ConfigDecorator} from 'story-utils/decorators';
 
-import AddChildModal from './AddChildModal';
+import {AddChildModal} from './AddChildModal';
```

Comment:
why the import change?

Reply:
yeah, in general using the default export makes it a bit more clear what the public API is